activejdbc/src/test/java/org/javalite/activejdbc/ModelTest.java

```diff
@@ -521,18 +522,18 @@ public void testDeleteCascade(){
 
     @Test
     public void shouldGenerateCorrectInsertSQL() {
-        Student s = new Student();
-        s.set("first_name", "Jim");
-        s.set("last_name", "Cary");
-        s.set("id", 1);
-        String insertSQL = s.toInsert();
-        // date literals formatting is not the same for every DBMS, so not testing this here
-        the(insertSQL).shouldBeEqual("INSERT INTO students (first_name, id, last_name) VALUES ('Jim', 1, 'Cary')");
+        Address address = new Address();
```

Comment:
what is the reason to switch to a different model?

Reply:
The reason I've changed the Model was because `Students` has a date, and that type of field causes this:

```java
FAILURE! - in org.javalite.activejdbc.ModelTest
shouldGenerateCorrectInsertSQLWithReplacements(org.javalite.activejdbc.ModelTest)  Time elapsed: 0.01 sec  <<< ERROR!
org.javalite.test.jspec.TestException: 
Test object:
java.lang.String == <INSERT INTO students (dob, first_name, last_name) VALUES (CONVERT(date, '1965-12-01'), 'Jim', 'O''Connor''s')>
and expected
java.lang.String == <INSERT INTO students (dob, first_name, last_name) VALUES (DATE '1965-12-01', 'Jim', 'O''Connor''s')>
are not equal, but they should be.
	at org.javalite.activejdbc.ModelTest.shouldGenerateCorrectInsertSQLWithReplacements(ModelTest.java:552
```
> http://ci.javalite.io/job/activejdbc-mssql/126/consoleFull

An insert statement that is DBMS dependent 😞 

Seeing this error, I though of 2 options:
* Somehow retrieving the DATE format from the Dialect
* Changing to another model without a DATE

Looking at the test I though the use of DATE wasn't all that relevant, and so I've changed the Model to another one without this type...

Should I have opted for another option?
Thanks

Reply:
No, I think this is OK, making sense. What about my other comment?